.github/workflows/ci.yaml

```diff
@@ -0,0 +1,69 @@
+name: CI
```

Comment:
Oh, I changed it because I though `integrate.yaml` sounded like it was only doing integration testing, rather than the whole thing, but if you've decided, I won't open this debate again. I though `ci.yaml` was more appropriate because that is exactly what that workflow is doing.